projects/core/src/icon/icon.element.spec.ts

```diff
@@ -94,6 +95,31 @@ describe('icon element', () => {
       expect(component.requestUpdate).not.toHaveBeenCalled();
       expect(component.getAttribute('shape')).toEqual(testShape);
     });
+
+    it('does not subscribe to the global state handler after the element is disconnected', async () => {
+      const spy = spyOn(GlobalStateService.stateUpdates, 'subscribe');
+
+      testElement = await createTestElement(html`<cds-icon shape="testing"></cds-icon>`);
+      expect(GlobalStateService.stateUpdates.subscribe).toHaveBeenCalledTimes(1);
+
+      spy.calls.reset();
+
+      // https://github.com/vmware-clarity/core/issues/61
+      // verify the element is not subscribed when it is removed before connecting
+      const div = document.createElement('div');
+      document.body.appendChild(div);
+
+      const icon = document.createElement('cds-icon');
+      icon.setAttribute('shape', 'testing');
+      div.appendChild(icon);
+      icon.remove();
+
+      await new Promise(resolve => setTimeout(resolve, 0));
```

Comment:
this is rough, but I don't have a web component to wait on since I'm explicitly removing it immediately above.